src/maliput_viz/plugins/maliput_mesh_converter.cc

```diff
@@ -0,0 +1,189 @@
+      [center](const std::tuple<ignition::math::Vector3d, int>& a, const std::tuple<ignition::math::Vector3d, int>& b) {
+        double a_y = std::atan2(std::get<0>(a).Y() - center.Y(), std::get<0>(a).X() - center.X());
+        double b_y = std::atan2(std::get<0>(b).Y() - center.Y(), std::get<0>(b).X() - center.X());
+        if (a_y < 0) a_y += 2 * M_PI;
+        if (b_y < 0) b_y += 2 * M_PI;
+        return a_y < b_y;
+      });
+  return ordered_vector;
+}
+
+// \brief Generates a unique name for a mesh.
+// \param baseName The base name. It will be appended with "_ID".
+// \return A string formatted like: @p baseName + "_" + ID , where ID is a
+// increasing unsigned integer.
+const std::string GenerateUniqueMeshName(const std::string& baseName) {
+  static uint counter = 0;
+  return baseName + "_" + std::to_string(counter++);
+}
+
+}  // namespace
+
+std::unique_ptr<ignition::common::Mesh> Convert(const std::string& name, const GeoMesh& geo_mesh) {
+  // Checks before actually creating the mesh.
+  if (geo_mesh.num_vertices() < 3 && geo_mesh.faces().size() < 1) {
+    return nullptr;
+  }
+
+  auto mesh = std::make_unique<ignition::common::Mesh>();
+  // TODO(agalbachicar):    ignition::rendering and ignition::common do not
+  //                        support unloading meshes. So, as for now, we need to
+  //                        create meshes with unique names (by adding an
+  //                        increasing ID to name) so it can be referenced and
+  //                        not mixed with another mesh.
+  //                        This ticket has been created to track this issue:
+  // https://bitbucket.org/ignitionrobotics/ign-rendering/issues/27/mesh-addition-removal-and-addition-with
+  mesh->SetName(GenerateUniqueMeshName(name));
+
+  auto sub_mesh = std::make_unique<ignition::common::SubMesh>();
+  sub_mesh->SetPrimitiveType(ignition::common::SubMesh::TRIANGLES);
+
+  auto geo_position_to_ign_vector = [](const maliput::api::InertialPosition& v) {
+    return ignition::math::Vector3d(v.x(), v.y(), v.z());
+  };
+
+  // Adds vertices, fake normals and fake texture coordinates. Each vertex needs
+  // a normal and a texture coordinate. Fake texture coordinates are added given
+  // that Drake's meshes do not have any information about that and normals are
+  // filled with stubs to match the same number of vertices. Later, when
+  // iterating through faces, normals will be filled with the correct value.
+  // Note that geo_mesh may not have the same number of vertices as normals
+  // given that it keeps no duplicates.
+  for (int i = 0; i < geo_mesh.num_vertices(); ++i) {
+    sub_mesh->AddVertex(geo_position_to_ign_vector(geo_mesh.get_vertex(i).v()));
+    sub_mesh->AddNormal({0, 0, 1});
+    sub_mesh->AddTexCoord({0, 0});
+  }
+
+  // Sets the indices based on how the faces were built.
+  for (const IndexFace& index_face : geo_mesh.faces()) {
+    // TODO(agalbachicar):    I'm assuming that IndexFace will not have more
```

Comment:
Shall we create / migrate a ticket for this TODO? Is it still relevant?

Reply:
No need for a ticket/todo but I left it as documentation of the behavior.